lib/cosmos/system/system.rb

```diff
@@ -60,7 +60,7 @@ class System
     instance_attr_reader :limits_set
 
     # Known COSMOS ports
-    KNOWN_PORTS = ['CTS_API', 'TLMVIEWER_API', 'CTS_PREIDENTIFIED']
+    KNOWN_PORTS = ['CTS_API', 'TLMVIEWER_API', 'CTS_PREIDENTIFIED', 'CTS_CMD_ROUTER']
```

Comment:
So CTS_PREIDENTIFIED is the tlm router and CTS_CMD_ROUTER is the cmd router. Maybe we can make an alias for tlm router as CTS_TLM_ROUTER? CTS_PREIDENTIFIED has always been a bit of a mystery to people. I know why you named it that because you preidentify the packets with tgt & pkt name but it doesn't mean much to most people.